lesson07/assignment/README.md

```diff
@@ -15,79 +15,35 @@ In the last assignment in lesson06, you created a responsive website with Bootst
-	- 	Installing scss (in your project directory)
-		-	npm command: run `npm install -D sass-loader sass`
-		-	yarn command: run `yarn add sass-loader sass`
-	-	Installing Bootstrap (in your project directory)
-		-	npm command: run `npm install bootstrap-vue bootstrap jquery popper.js`
-		-	yarn command: run `yarn add bootstrap-vue bootstrap jquery popper.js`
-	-	Adding bootstrap/scss to the project
-		-	In your `src` folder, create a `main.scss` file
-		-	In that new file, add the following:
-			```
-			@import 'node_modules/bootstrap/scss/bootstrap';
-			@import 'node_modules/bootstrap-vue/src/index.scss';
-			```
-		-	You can create & import partials into this file or add additional global styles using scss
-		-	Also in your `src` folder, open `main.js` and add the following under the router import:
-			```
-			import 'bootstrap'
-			import BootstrapVue from 'bootstrap-vue'
-			import './main.scss'
 
-			Vue.use(BootstrapVue)
-			```
-		-	You now have access to scss and all of bootstrap/bootstrap-vue. Watch the screencast for more information on using bootstrap-vue components and bootstrap features in Vue, as well as turning the bootstrap nav into an app router
-	-	To run the project, `cd` into your project and run `npm run dev` or `yarn dev` to launch the server
+###	Create your project using Vue CLI:
 
--	To create your project using Vue CLI v3,
-	-	run `vue create my-title`
-	-	Choose 1st option (default)
-	-	Once done, `cd` into your project
-	-	run `vue add router`
-		-	Select `Y` for history
-		-	You'll see the router files were generated for you
-	- 	Installing scss (in your project directory)
-		-	npm command: run `npm install -D sass-loader sass`
-		-	yarn command: run `yarn add sass-loader sass`
-	-	Installing Bootstrap (in your project directory)
-		-	npm command: run `npm install bootstrap-vue bootstrap jquery popper.js`
-		-	yarn command: run `yarn add bootstrap-vue bootstrap jquery popper.js`
-	-	Adding bootstrap/scss to the project
-		-	In your `src` folder, create a `main.scss` file
-		-	In that new file, add the following:
-			```
-			@import 'node_modules/bootstrap/scss/bootstrap';
-			@import 'node_modules/bootstrap-vue/src/index.scss';
-			```
-		-	You can create & import partials into this file or add additional global styles using scss
-		-	Also in your `src` folder, open `main.js` and add the following under the router import:
-			```
-			import 'bootstrap'
-			import BootstrapVue from 'bootstrap-vue'
-			import './main.scss'
+-	Follow the instructions in the Vue CLI documentation: https://cli.vuejs.org/guide/creating-a-project.html. Choose "default" preset, rather than choosing features manually. If you're not sure whether to use npm or yarn, choose npm. Pass the "--no-git" flag to the create command, to prevent it from initializing the new project as a git repository (necessary because the html300 directory is a repository already, and having nested repositories leads to major problems with git).
```

Comment:
I think the "--no-git" should be mentioned sooner, around where we tell them to use the vuejs documentation. Since you attach it to the vue create command, I hadn't realized I needed to add it until I selected npm.

Reply:
👍